client/src/components/StaticPagesMCK/About.js

```diff
@@ -3,300 +3,422 @@ import React from "react";
 import Container from "@mui/material/Container";
 import Typography from "@mui/material/Typography";
+import { PageWrapper } from "components/StaticPages/PageWrapper";
+import { Link as RouterLink } from "react-router-dom";
+import { Box, CardMedia, Link, SvgIcon } from "@mui/material";
+import IconSpacerSVG from "components/StaticPages/assets/IconSpacerSVG";
+import Footer from "../Layout/Footer";
 
-const useStyles = makeStyles(() => ({
-  outer: {
-    background: "#fff",
-  },
-  main: {
-    padding: "1.5rem 0;",
-    maxWidth: "1200px",
-    margin: "0 auto",
-    "@media only screen and (min-width: 75em)": {
-      padding: "1.5rem 2rem",
-    },
-  },
-  title: {
-    textTransform: "uppercase",
-    fontWeight: 500,
-    textAlign: "center",
-    background: "#FFF",
-    margin: 0,
-    padding: "32px 0",
-  },
-  figure: {
-    margin: 0,
-    padding: 0,
-  },
-  icon: {
-    margin: "auto",
-  },
-  mission: {
-    padding: "32px",
-    margin: "32px 0",
-    borderRadius: "24px",
-    background: "#f0f0f0",
-    display: "flex",
-    flexDirection: "column",
-    "& $h2": {
-      flexBasis: "100",
-      textAlign: "center",
-      fontWeight: "500",
-      fontSize: "32px",
-      marginTop: "15px",
-      marginBottom: "15px",
-    },
-    "& $p": {
-      marginBottom: "16px",
-    },
-  },
-  team: {
-    padding: "32px",
-    margin: "32px 0 0 0",
-    borderRadius: "24px",
-    background: "#B6D8FB",
-    display: "flex",
-    flexDirection: "column",
-    "& $h2": {
-      flexBasis: "100",
-      textAlign: "center",
-      fontWeight: "500",
-      fontSize: "32px",
-      marginTop: "15px",
-      marginBottom: "15px",
-    },
-    "& $p": {
-      marginBottom: "16px",
-    },
-  },
-  contact: {
-    padding: "32px",
-    margin: "32px 0 0 0",
-    borderRadius: "24px",
-    background: "#f0f0f0",
-    display: "flex",
-    flexDirection: "column",
-    textAlign: "center",
-    "& $h2": {
-      flexBasis: "100",
-      textAlign: "center",
-      fontWeight: "500",
-      fontSize: "32px",
-      marginTop: "15px",
-      marginBottom: "15px",
-    },
-  },
-  partners: {
-    background: "#fff",
-    padding: "48px 32px",
-    borderRadius: "24px",
-    justifyContent: "space-between",
-    alignItems: "center",
-    display: "flex",
-    flexWrap: "wrap",
-    flexDirection: "column",
-    "& $h2": {
-      width: "100%",
-      flexBasis: "100",
-      textAlign: "center",
-      fontWeight: "500",
-      fontSize: "32px",
-      marginTop: "10px",
-      marginBottom: "60px",
-    },
-    "@media only screen and (min-width: 64em)": {
-      flexDirection: "row",
-    },
-  },
-  partnersLogo: {
-    maxWidth: "100%",
-    height: "100%",
-    marginBottom: "40px",
-    "@media only screen and (min-width: 64em) ": {
-      marginBottom: 0,
-    },
-  },
-}));
 const About = () => {
-  const classes = useStyles();
   // const { t } = useTranslation("about");
```

Comment:
I think we can remove this if it's not being used 🤔

Reply:
fixed